src/main/java/org/jetbrains/research/anticopypaster/ide/RefactoringNotificationTask.java

```diff
@@ -58,7 +58,7 @@ private PredictionModel getOrInitModel() {
             this.metricsGatherer = new MetricsGatherer();
```

Comment:
If we're adding in the actual model it probably wouldn't hurt to get rid of this line, since the first line of the UserSettingsModel should probably be making the MetricsGatherer.

Reply:
This has been pushed out to the model class